docs/application/dotnet/guides/nui/layouts.md

```diff
@@ -0,0 +1,249 @@
+# Layouts
+
+The Layout framework enables [View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) to be automatically positioned within a parent `View` that has been assigned a layout.
+The framework provides a common method to layout **Views** with just minimal setup in the application.
+
+Especially, NUI Layouts provide advanced reusable laying out capabilities. These capabilities separate the layout logic from Views into distinct layouts so that `View`'s responsibility is reduced to building and rendering the content.
+
+A `View` set by one layout can include an additional `View` set by another layout. For example, the following image shows `View` with a horizontal layout, which has a child `View` with the vertical layout:
+
+![NUILayouts](./media/NUILayoutImage.png)
+
+The default NUI layouts are [Linear Layout](./linear-layout.md), [Grid Layout](./grid-layout.md), and [Flex Layout](./flex-layout.md).
+When the default layouts are assigned to `View`, the children added to this `View` are positioned and sized according to the layout. The specification and properties set on the parent `View` also affect the position and the size of children.
+
+In addition, the layout framework allows nested layouts. `View` with a layout can be added to another `View` with a layout.
+
+A `View` can be assigned to any layout and then can be re-assigned to a different layout. However, a layout can be used with one `View` at a time.
+
+New layouts can be implemented by deriving them from the layout base class and positioning the children according to the requirement. To create a custom layout, see [Creating custom Layouts](./custom-layout.md).
+
+
+## How to Lay out View
```

Comment:
Suggestion:

Since we use title case for all the heading. Here "out" should also be written as "Out".

```suggestion
## How to Lay Out View
```

Reply:
Okay, Done.